src/person/serializers.py

```diff
@@ -62,8 +62,6 @@ def get_rides(self, user):
         for ride in rides:
             if ride.departure_datetime >= timezone.now():
                 active_rides.add(ride)
-            else:
-                ride.delete()
```

Comment:
is this different than what noah had deleted in his PR?

Reply:
it should be the same